docs/source/notes/randomness.rst

```diff
@@ -121,3 +131,22 @@ CUDA RNN and LSTM
 In some versions of CUDA, RNNs and LSTM networks may have non-deterministic behavior.
 See :meth:`torch.nn.RNN` and :meth:`torch.nn.LSTM` for details and workarounds.
 
+DataLoader
+..........
+
+DataLoader will reseed workers following :ref:`data-loading-randomness` algorithm.
+Use :meth:`worker_init_fn` to preserve reproducibility::
```

Comment:
"preserve reproducibility" -> "to set the seed(s) on each worker"

Would that be more direct? Should this mention why the worker_seed is computed as it is?